pkgs/development/libraries/glm/default.nix

```
@@ -28,10 +28,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  env.NIX_CFLAGS_COMPILE =
-    lib.optionals (stdenv.cc.isGNU && lib.versionAtLeast stdenv.cc.version "11") [
-      "-fno-ipa-modref" # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=102823
-    ];
+  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=102823
+  NIX_CFLAGS_COMPILE = lib.optionalString (stdenv.cc.isGNU && lib.versionAtLeast stdenv.cc.version "11") "-fno-ipa-modref";
```

```suggestion
  env.NIX_CFLAGS_COMPILE = lib.optionalString (stdenv.cc.isGNU && lib.versionAtLeast stdenv.cc.version "11") "-fno-ipa-modref";
```
See https://github.com/NixOS/nixpkgs/pull/217206/commits/f9fdf2d4028eac28a074c9ad75d309b3dcbf8e7a.

---

Whoops, that's what I meant to do.